runner.yml.dist

```diff
@@ -8,7 +8,7 @@ drupal:
     user: "root"
     password: ""
   post_install:
-    - "./vendor/bin/drush en oe_theme_test -y"
+    - "./vendor/bin/drush en oe_theme_demo -y"
```

Comment:
The `oe_theme_demo` module has configuration that requires the `oe_theme` theme to be already installed, so this line should be moved after line 14.